sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10071,21 +10071,21 @@
               "breadcrumb": "default",
               "id": "ef3gfa",
               "addons": [],
-              "lazyLoad": false
+              "lazyLoad": false,
+              "tags": []
             },
             {
-              "label": "<strong>Were you ever a youth in continuing care or custody of a director of child welfare in BC (ward of the court – this means the provincial government is/was your legal guardian)? </strong>",
+              "label": "<strong>Were you ever considered a child or youth under government care?</strong>",
```

Comment:
Thanks for stating the workflow reference in the PR description. 

![image](https://github.com/bcgov/SIMS/assets/54600590/9a63716c-6b21-42c0-a6a4-881f69d6d3c4)

I have a question here to business. 

Should the condition in workflow change from `studentDataYouthInCare === "yes"` to `(studentDataYouthInCare === "yes" && studentDataEverInCustodyOfChildWelfare === "yes")`  ?

Reply:
no, as mentioned in the PR description, for the above logics, the parent question is to be considered,, That's what Hugh told me.

Reply:
Checked with @HRAGANBC during development and the `studentDataEverInCustodyOfChildWelfare` should not affect the workflow. The checks still must happen based on `studentDataYouthInCare`.

Reply:
Thanks for the clarification